AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -42,6 +42,8 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 skay.ua##.newsletter_block_section
 levi.com##div[data-testid="auth-modal-controller"]
 tomshardware.com###slice-container-exitIntent
+lamartina.com#$#.MuiDialog-root[role="presentation"][style^="position: fixed; z-index: "] { display: none !important; }
```

Comment:
This rule also blocks a useful popup
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/176673247-fa3d24eb-b3da-4e82-a259-5c8abd700969.png)

</details><br/>